react/src/components/elements/plan-manager/CheckoutForm.tsx

```diff
@@ -0,0 +1,142 @@
+import { useState } from "react";
+import {
+  LinkAuthenticationElement,
+  PaymentElement,
+} from "@stripe/react-stripe-js";
+import { useStripe, useElements } from "@stripe/react-stripe-js";
+import { Box, Flex } from "../../ui";
+
+export const CheckoutForm = () => {
+  const stripe = useStripe();
+  const elements = useElements();
+
+  const [message, setMessage] = useState<string | null>(null);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleSubmit: React.FormEventHandler<HTMLFormElement> = async (
+    event,
+  ) => {
+    event.preventDefault();
+
+    if (!stripe || !elements) {
+      // Stripe.js has not yet loaded.
+      // Make sure to disable form submission until Stripe.js has loaded.
+      return;
+    }
+
+    setIsLoading(true);
+
+    const { error } = await stripe.confirmPayment({
+      elements,
+      confirmParams: {
+        // Make sure to change this to your payment completion page
+        return_url: `${window.location.origin}/completion`,
```

Comment:
What is this /completion URL?

Reply:
Left over from Stripe's example. 😆 I updated this to use the current url.